python/spear/config/default_config.spear.yaml

```diff
@@ -4,81 +4,96 @@
+  INSTANCE:
+
+    # Launch Unreal application via different modes:
+    # - "standalone" launches a packaged executable of an Unreal Engine project
+    # - "uproject" launches an uproject with Unreal Engine editor executable
+    # - "none" connects to an already running instance of Unreal Engine
+    LAUNCH_MODE: ""
+
+    # If launch_mode is "standalone_exectuable", specify this path to an Unreal application.
+    STANDALONE: ""
+
+    # If launch_mode is "uproject", specify this path to your Unreal Engine editor executable.
+    EDITOR_EXECUTABLE: ""
+
+    # If launch_mode is "uproject", specify this path to your uproject file.
+    UPROJECT: ""
+
+    # The location of downloaded (or built) pak files. This must be specified if you are attempting
+    # to load any scene other than the default scene.
+    PAKS_DIR: ""
+
+    # Description of certain parameters
+    # -> resx and resy
+    #     If not running in headless mode, these parameters specify the window resolution used for
+    #     on-screen rendering. Has no effect on the size of the visual observations returned by the
+    #     application via the RPC server.
+    # -> graphicsadapter
+    #     Launch your Unreal application on the GPU specified by GPU_ID.
+    # -> log
+    #     Log file to capture Unreal debug output that isn't captured by STDOUT and STDERR. This is
+    #     useful for inspecting the stack trace when an Unreal application crashes. Note that Unreal
+    #     only writes to this log file in Development builds. If a path is included, it will be
+    #     interpreted as a relative path regardless of how it is formatted, and will be interpreted
+    #     as relative to the following locations:
+    #     - Windows: path\to\spear\cpp\unreal_projects\SpearSim\Standalone-Development\Windows\SpearSim\Saved\Logs
+    #     - macOS:   ~/Library/Logs/SpearSim
+    #     - Linux:   path\to\spear\cpp\unreal_projects\SpearSim\Standalone-Development/Linux/SpearSim/Saved/Logs
+    # -> renderoffscreen
+    #     Set this parameter to true True to launch the executable in headless mode. Has no effect if launch_mode is "none".
+    COMMAND_LINE_ARGS:
+      resx: 1024
+      resy: 1024
+      graphicsadapter: 0
+      log: ""
+      windowed: null
+      nosound: null
+      fileopenlog: null
+      stdout: null
+      fullstdoutlogoutput: null
+      nologtimes: null
+      renderoffscreen: False
+
+    # Path to a temp dir for files generated by the spear Python package
+    TEMP_DIR: "tmp"
+
+    # Name for a temp yaml config file generated by the spear Python package
+    TEMP_CONFIG_FILE: "config.yaml"
+
+    # List of custom command-line arguments passed to the Unreal application. This is useful
+    # when the Unreal application is built with a plugin that expects its own set of
+    # command-line arguments.
+    CUSTOM_COMMAND_LINE_ARGUMENTS: []
+
+    # This parameter can be used for additional control over which Vulkan devices are recognized
```

Comment:
Let's leave this comment because it's a useful reminder, but let's move this to a generic dictionary called `SPEAR.ENVIRONMENT_VARS`, similar to `SPEAR.INSTANCE.COMMAND_LINE_ARGS`.